### PR TITLE
Feature/notification of report

### LINKED ISSUE #172: Teacher is notified during a sequence when an evaluation is reported
### Description

As a teacher connected to the A sequence,
I want to be notified when a student reported an evaluation in the A sequence
So that I can quickly react to the situation  

### Acceptance criteria

- [ ] The teacher is notified directly during the sequence (s)he is connected to inside the elaastic interface.

### PR DESCRIPTION
**Pull request by Francois389**
close #172 

# Result

image

### CODE REVIEW

File: src/main/kotlin/org/elaastic/questions/player/components/sequenceInfo/SequenceInfoResolver.kt [hunk 1]
```diff
@@ -23,95 +23,116 @@ import org.elaastic.questions.assignment.sequence.Sequence
 import org.elaastic.questions.assignment.sequence.State
 import org.elaastic.questions.assignment.sequence.interaction.InteractionType
 import org.elaastic.questions.controller.MessageBuilder
-import java.lang.IllegalStateException
+import org.springframework.stereotype.Component
```
Comment (FranckSilvestre) [inline src/main/kotlin/org/elaastic/questions/player/components/sequenceInfo/SequenceInfoResolver.kt:26]: not used

File: src/main/kotlin/org/elaastic/questions/assignment/sequence/peergrading/PeerGradingService.kt [hunk 2]
```diff
@@ -98,6 +98,10 @@ class PeerGradingService(
     fun findAllDraxo(response: Response): List<DraxoPeerGrading> =
         peerGradingRepository.findAllByResponseAndType(response, PeerGradingType.DRAXO)
 
+    fun findAllDraxo(sequence: Sequence): List<DraxoPeerGrading> =
+        findAll(sequence)
+            .filterIsInstance<DraxoPeerGrading>()
+
```
Comment (FranckSilvestre) [inline src/main/kotlin/org/elaastic/questions/assignment/sequence/peergrading/PeerGradingService.kt:104]: I don't understand the filter

**Review by FranckSilvestre — CHANGES_REQUESTED** [review]

File: src/main/kotlin/org/elaastic/questions/assignment/sequence/peergrading/PeerGradingService.kt (hunk 2 above)
Comment (Francois389) [reply src/main/kotlin/org/elaastic/questions/assignment/sequence/peergrading/PeerGradingService.kt:104]: With this method, I just want the DRAXO peer grading.

`DraxoPeerGrading` inherit of `PeerGrading` and the `findAll` method return only `PeerGrading` object. So to ensure I only return `DraxoPeerGrading` I use this filter. 
I could use this filter instead : 
```kotlin
.filter { it is DraxoPeerGrading }
```
But this filter return me a List of PeerGrading (`List<PeerGrading>`) wich will force me to cast it.
The filter I use return me a List of DraxoPeerGrading (`List<DraxoPeerGrading>`). It's more efficient and easier to read.

File: src/main/kotlin/org/elaastic/questions/player/components/sequenceInfo/SequenceInfoResolver.kt (hunk 1 above)
Comment (Francois389) [reply src/main/kotlin/org/elaastic/questions/player/components/sequenceInfo/SequenceInfoResolver.kt]: Fix in 3b399d4871ea7e8964b875d608b780eb604afb92

**Review by FranckSilvestre — APPROVED** [review]

**Pull request merged**